test/integration/suite_test.go

```diff
@@ -42,6 +44,9 @@ const gatewayAPIsCRDs = "https://github.com/kubernetes-sigs/gateway-api.git/conf
 var (
 	existingClusterName  = os.Getenv("KONG_TEST_CLUSTER")
 	controllerManagerOut = os.Getenv("KONG_CONTROLLER_OUT")
+	runWebhookTests      = false
+	webhookCertDir       = ""
+	webhookServerIP      = ""
```

Comment:
Why not pull this directly from the env?

```suggestion
	webhookServerIP      = os.Getenv("GATEWAY_OPERATOR_WEBHOOK_IP")
```

Reply:
updated